### PR TITLE
Prepare 2.3.0 release

### PR DESCRIPTION
**Pull request by no-reply**
Draft release notes are: https://github.com/samvera/hyrax/releases/tag/untagged-e5d52ca19728329f7b59

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by no-reply** [comment]
Testing carried out, and release notes written by @chrisdaaz.

We'll probably announce this release as a significant accessibility milestone, since it closes the remaining `a11y` audit issues that are rooted in Hyrax. Hopefully tickets that remain open can be fixed upstream, so adopters can upgrade without further Hyrax releases.

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ gemspec
 
 group :development, :test do
   gem 'coveralls', require: false
-  gem 'i18n-tasks'
+  gem 'i18n-tasks', '0.9.21'
```
Comment (jcoyne) [inline Gemfile]: Why such pessimism in versions?

Comment (no-reply) [reply Gemfile]: The new version of `i18n-tasks` breaks our current translation process. This is the version we are using now. Locking to it buys us time to update the process.

Note this only impacts development users trying to run translations through the google translate API.

In other words, is it really pessimism if the next version is actually broken? :)

Comment (jcoyne) [reply Gemfile]: So, `.22`, `.23`, & `.24` are all broken?  Can you add a comment to that effect here?

**Comment by no-reply** [comment]
@jcoyne I ended up updating the documentation to work with the latest releases. https://github.com/samvera/hyrax/wiki/Translations-(internationalization)

**Pull request merged**